packages/sdk/src/schema/pages.ts

```diff
@@ -43,7 +43,7 @@ const commonPageFields = {
   meta: z.object({
     description: z.string().optional(),
     title: z.string().optional(),
-    excludePageFromSearch: z.boolean().optional(),
+    excludePageFromSearch: z.string().optional(),
```

Comment:
why is that a string now?

Reply:
because its an expression which is a string? basically we don't know the resulting value type